test/e2e/tests/bridge/bridge-test-utils.ts

```diff
@@ -120,7 +122,7 @@ export const getBridgeFixtures = (
 
   return {
     driverOptions: {
-      openDevToolsForTabs: true,
+      // openDevToolsForTabs: true,
```

Comment:
Did you intend to delete this line?

Reply:
Thanks for taking a look! This is useful for local debugging, but it makes the tests flaky in CI so I commented it out